app/__init__.py

```diff
@@ -108,6 +108,7 @@ def create_app():
     # setup celery
     app.config['CELERY_BROKER_URL'] = app.config['REDIS_URL']
     app.config['CELERY_RESULT_BACKEND'] = app.config['CELERY_BROKER_URL']
+    app.config['CELERY_ACCEPT_CONTENT'] = ['pickle', 'json', 'application/text']
```

Comment:
Bad idea. This will suppress the warning. Just allow json and text and see if we get any errors. If we are still using pickle, port it to JSON